oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -249,13 +256,17 @@ impl WasmInterface {
 
         let write_handle = self.allocate_new_handle(writer, HandleDirection::Write);
         let read_handle = self.allocate_new_handle(reader, HandleDirection::Read);
+        debug!(
+            "{}: channel_create() -> ({}, {})",
+            self.pretty_name, write_handle, read_handle
+        );
 
         self.get_memory()
             .set_value(write_addr, write_handle as i64)
             .map_err(|err| {
                 error!(
-                    "channel_create: Unable to write writer handle into guest memory: {:?}",
-                    err
+                    "{}: channel_create(): Unable to write writer handle into guest memory: {:?}",
+                    self.pretty_name, err
```

Comment:
This seems quite annoying to maintain; could / should we have a node_{info,error,...} macro that just uses the correct prefix? We can make `Node` impl `Display` or `Debug` if it does not already, and the macro would just print `self` in that case. Can be a separate PR, if you think it's a good idea.